### ISSUE TITLE
[Accessibility] End alt text with a period

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/docs#16854]
Based on <https://www.w3.org/WAI/GL/techniques.htm>:

> "alt" - should be used to give the functional and contextualized label or description (rather than a visual description).   Think of it as a textual replacement for users and search engines.  It should not bring to the attention of the user that a picture exists and should be grammatically correct.  **End alt-text with punctuation (comma, colon, or period) to indicate a pause in reading, when appropriate**.

**Comment by mairaw** [comment]
Thanks @Youssef1313. I'm checking with the team what's the guidance on this and if we can get automatic validation for these cases.
